### PR TITLE
simplify common_camera_process_front

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

File: selfdrive/camerad/cameras/camera_common.cc [hunk 1]
```diff
@@ -345,58 +345,46 @@ std::thread start_process_thread(MultiCameraState *cameras, const char *tname,
...
-      meteringbox_xmax = rhd_front ? b->rgb_width * 2 / 5:b->rgb_width;
-    }
-  }
-
   // auto exposure
   if (cnt % 3 == 0) {
-    // use driver face crop for AE
-    int x_start, x_end, y_start, y_end;
-    int skip = 1;
+    if (sm->update(0) > 0) {
```
Comment (adeebshihadeh) [inline selfdrive/camerad/cameras/camera_common.cc:353]: explicitly check if driverState was updated, otherwise this breaks if more services are added

**Comment by ZwX1616** [comment]
cool! can you do a rebase on latest master?

**Comment by deanlee** [comment]
> cool! can you do a rebase on latest master?

done

**Comment by ZwX1616** [comment]
LGTM. Congrats on reaching your 100th openpilot PR!

**Comment by deanlee** [comment]
Thanks, I didn't even realize it was 100. a memorable moment.😄

**Pull request merged**